### DISCUSSION TITLE
[Feature request] Github Copilot should show suggestions to further code as i manually change code

### DISCUSSION THREAD

**Discussion by Mindaugas3** [community/community#114312] [Copilot Conversations]
### Select Topic Area

General

### Body

Github Copilot VSCode extension should show suggestions as I repeatedly rename things like variables in my code, as I type. Would be much more bang for my annual fee bucks, and make coding even faster.

**Comment by ghostinhershell** [comment]
Hi @Mindaugas3, 

Thanks so much for reaching out in our community 👋🏾 

I would like to share this feedback with the team but can you share more info on what you mean here and what this feature should look like?

**Reply by Mindaugas3** [reply]
For example if i have copy pasted a list it's like
`
1 item
1 item
1 item
`
it could suggest1 2 3 4 order, same with named variables or array items i might want to plug in

**Comment by vishal11u** [comment]
Response #1
